src/main/java/org/jenkinsci/plugins/builduser/varsetter/impl/TimerTriggerCauseDeterminant.java

```diff
@@ -0,0 +1,30 @@
+package org.jenkinsci.plugins.builduser.varsetter.impl;
+
+import java.util.Map;
+
+import org.jenkinsci.plugins.builduser.utils.UsernameUtils;
+import org.jenkinsci.plugins.builduser.varsetter.IUsernameSettable;
+
+import hudson.triggers.TimerTrigger;
+
+public class TimerTriggerCauseDeterminant implements IUsernameSettable<TimerTrigger.TimerTriggerCause> {
+
+	private static final String TIMER_TRIGGER_DUMMY_USER_NAME = "Timer Trigger";
+	private static final String TIMER_TRIGGER_DUMMY_USER_ID = "timer";
+
+    @Override
```

Comment:
It's fine for me. Just wanted to say that `@Override`is formatted differently being compared to the code nearby